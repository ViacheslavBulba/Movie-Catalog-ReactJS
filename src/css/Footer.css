.footer-container {
    width: 100vw;
    height: 70px;
    display: flex;
    flex-direction: column;
    align-items: center;
    background-color: #424242;
}

.footer-logo {
    width: 100px;
    height: 100%;
}
