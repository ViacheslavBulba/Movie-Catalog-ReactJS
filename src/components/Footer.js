import React from 'react';
import '../css/Footer.css';
import logo from '../../public/netflix-logo.svg';

export default function Footer() {

    return (
        <footer className="footer-container">
            <img
                src={logo}
                alt="footer logo"
                className="footer-logo"
            />
        </footer>
    )
}
