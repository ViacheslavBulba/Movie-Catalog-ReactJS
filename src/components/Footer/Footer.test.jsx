import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './Footer';

test('snapshot test for footer', () => {
  const component = renderer.create(
    <Footer />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
