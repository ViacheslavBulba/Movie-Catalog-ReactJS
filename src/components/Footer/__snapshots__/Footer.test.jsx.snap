// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for footer 1`] = `
<footer
  className="footer-container"
>
  <img
    alt="footer logo"
    className="footer-logo"
    src="netflix-logo.svg"
  />
</footer>
`;
